docs/userguide/ui-concepts.md

```diff
@@ -1,3 +1,151 @@
 # Workflow UI
 
-_Coming soon!_
+This page provides a high level overview of Workflow UI, the companion that allows [Workflow Core](../core-workflow) to drive Android and iOS apps.
+
+## Separation of Concerns
+
+Workflow maintains a rigid separation between its core runtime and its UI support.
+The [Workflow Core](../core-workflow) modules are strictly Swift and Kotlin, with no dependencies on any UI framework.
+Those dependencies are restricted to the Workflow UI modules, as you would expect, with Swift UI relying on iOS, and Kotlin UI on Android.
+This innate separation naturally puts developers on a path to avoid entangling view concerns with their app logic.
+
+
+!!! info "Separate both ways"
+    This strictness goes in both directions: neither UI library depends upon either runtime. They could easily be driven by any other stream of view models, and in view system tests they often are.
+
+    The "rendering" terminology does show up in the UI API and docs, but strictly as a convention, to make it easier to understand where [Workflow Core](../core-workflow) and Workflow UI are designed to bolt together.
+
+
+And note that we say "app logic" rather than "business logic."
+In any interesting app, the code that manages navigation and other UI-releated behavior is likely to dwarf that for what we typically think of as model concerns, in both size and complexity. 
+
+We're all pretty good at capturing business concerns in tidy OO models of items for sale, shopping carts, payment cards and the like, nicely decoupled from the UI world.
+But the rest of the app, and in particular the bits about how our users navigate it? Traditionally it's hard to keep that app logic code centralized, so that you can see what's going on; and even harder to keep it decoupled from your view system, so that it's easy to test.
+The strict divide between Workflow UI and the runtime makes it easy.
+
+## What's a Screen?
+
+Most Workflow implementations produce pure-Swift / pure-Kotlin renderings that can serve as view models.
```

Comment:
What's a Rendering? Like, it'll be awesome to see how rendering and screen relate to each other, and what is the right way to call things depending on context.

Reply:
How about if I link to the glossary entry for that? Aaand of course there are no automatic bookmarks for DL entries, yay.

https://square.github.io/workflow/glossary/#:~:text=is%20strongly%20discouraged.-,Rendering,-The%20externally%20available

Reply:
Also note that rendering is already described in some depth in the core docs. Maybe should link to that instead? 

https://square.github.io/workflow/userguide/core-workflow/#workflows-produce-an-external-representation-of-their-state-via-rendering

Reply:
Sounds good, and we could explain that Screen could be a derivative of Rendering, tailored to be responsible for UI representation and interaction (while Rendering itself is kind of a public snapshot of the state; but snapshot, and not The Snapshot). Damn, workflow terminology already employed every possible term I could think of :)

Reply:
I'm wary of going too far into the weeds here.